### PR TITLE
Made a search-bar 

### PR DESCRIPTION
**Pull request by ia-lerimund**
Why does the program not see the bottom border of the second form in tests?

- [DEMO LINK](https://ia-lerimund.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://ia-lerimund.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,45 @@
...
+        <input 
+          class="search-bar small" 
+          type="text" 
+          name="search-bar-s" 
+          id="search-bar-s"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="small"
+        >
+      </label>
+    </form>
```
Comment (stanislav-topikha) [inline src/index.html:53]:
```suggestion
    <form 
      action="/" 
      method="get"
      data-qa="big"
    >
      <label class="label" for="search-bar-b">
        <img 
          class="search-icon" 
          src="./images/Search.svg" 
          alt="search-icon">
        <input 
          class="search-bar" 
          type="text" 
          name="search-bar-b" 
          id="search-bar-b"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>
    <form 
      action="/" 
      method="get"
      data-qa="small"
    >
      <label class="label" for="search-bar-s">
        <img 
          class="search-icon search-icon-small" 
          src="./images/Search.svg" 
          alt="search-icon">
        <input 
          class="search-bar search-bar-small" 
          type="text" 
          name="search-bar-s" 
          id="search-bar-s"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="small"
        >
      </label>
    </form>
```

lets make it work like this

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
remove the outline 
image

and comments ⬇️

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body form:first-child {
+  margin: 20px auto;
+}
```
Comment (stanislav-topikha) [inline src/style.css:28]: dont use tag selectors

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+.search-bar {
+  font-family: "Avenir", sans-serif;
```
Comment (stanislav-topikha) [inline src/style.css:31]: add fallback font, set it to `html`

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
image
image

links dosent work

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**